development-vm/replication/replicate-data-local.sh

```diff
@@ -12,15 +12,14 @@ bundle install --quiet
 
 $(dirname $0)/sync-aws-mysql.sh "$@" mysql-master
 
-#$(dirname $0)/sync-mongo.sh "$@" mongo-1.backend.integration
-#$(dirname $0)/sync-mongo.sh "$@" api-mongo-1.api.integration
-#$(dirname $0)/sync-mongo.sh "$@" router-backend-1.router.integration
-
-#if ! ($SKIP_MONGO || $DRY_RUN); then
-#  status "Munging router backend hostnames for dev VM"
-#  mongo --quiet --eval 'db = db.getSiblingDB("router"); db.backends.find().forEach( function(b) { b.backend_url = b.backend_url.replace(".integration.publishing.service.gov.uk", ".dev.gov.uk").replace("https","http"); db.backends.save(b); } );'
-#  mongo --quiet --eval 'db = db.getSiblingDB("draft_router"); db.backends.find().forEach( function(b) { b.backend_url = b.backend_url.replace(".integration.publishing.service.gov.uk", ".dev.gov.uk").replace("https","http"); db.backends.save(b); } );'
-#fi
+$(dirname $0)/sync-aws-mongo.sh "$@" mongo
+$(dirname $0)/sync-aws-mongo.sh "$@" router_backend
```

Comment:
It looks like we've gone from three MongoDBs to two. Is api-mongo no longer used, or is it included in one of the other database dumps?

Reply:
I believe it is included in one of the others, but will check. as we are not dumping it at the moment so that may be an oversight.